packages/three-vrm/src/index.ts

```diff
@@ -9,3 +9,6 @@ export * from '@pixiv/three-vrm-core';
 export * from '@pixiv/three-vrm-materials-mtoon';
 export * from '@pixiv/three-vrm-node-constraint';
 export * from '@pixiv/three-vrm-springbone';
+
+export type { VRMMaterialsV0CompatPlugin } from '@pixiv/three-vrm-materials-v0compat';
+export type { VRMNodeConstraintLoaderPlugin } from '@pixiv/three-vrm-node-constraint';
```

Comment:
Doesn't it overwrapped with the line `export * from '@pixiv/three-vrm-node-constraint';` ?